### PR TITLE
Add pagination on issues

### PR DESCRIPTION
**Pull request by stefanius**

### CODE REVIEW

File: src/Actions/ManagesIssues.php [hunk 1]
```diff
@@ -14,12 +14,19 @@ trait ManagesIssues
...
      */
-    public function issues($projectId)
+    public function issues($projectId, $limit = 25, $page = 1)
     {
-        $result = $this->get('issues', ['project_id' => $projectId]);
+        $result = $this->get('issues', ['query' => [
+            'project_id' => $projectId,
+            'page_size' => $limit,
+            'page' => $page,
+        ]]);
```
Comment (thijskok) [inline src/Actions/ManagesIssues.php]: Variable $project remains not renamed ⬇️

File: src/Actions/ManagesIssues.php [hunk 2]
```diff
@@ -14,12 +14,19 @@ trait ManagesIssues
...
      *
      * @param string $projectId
+     * @param int $limit
+     * @param int $page
      *
      * @return Issue[]
+     * @throws \TestMonitor\Mantis\Exceptions\InvalidDataException
      */
-    public function issues($projectId)
+    public function issues($projectId, $limit = 25, $page = 1)
```
Comment (thijskok) [inline src/Actions/ManagesIssues.php]: I'd use Mantis' default, limit = 50

Comment (thijskok) [reply src/Actions/ManagesIssues.php]: And we use strong typing in the other clients, perhaps here as well?

Comment (camlafit) [reply src/Actions/ManagesIssues.php]: If we follow strict API implementation in this case $projectId should be optional.

File: .gitignore [hunk 3]
```diff
@@ -10,3 +10,6 @@ phpunit.xml
 /build
 
 /.idea
+
+test.php
```
Comment (thijskok) [inline .gitignore]: Oops...

File: src/Actions/ManagesIssues.php [hunk 4]
```diff
@@ -13,18 +13,25 @@ trait ManagesIssues
     /**
      * Get a list of of issues.
      *
-     * @param string $projectId
+     * @param int $projectId
```
Comment (thijskok) [inline src/Actions/ManagesIssues.php]: int|null ?

File: src/Actions/ManagesIssues.php (hunk 2 above)
Comment (thijskok) [reply src/Actions/ManagesIssues.php]: Agreed

**Review by thijskok — APPROVED** [review]

**Pull request merged**